### ISSUE TITLE
Improve dialog framework

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#11513]
Better accessibility by returning focus, focus-trap including portals and more.
See ARIA examples for things to do

**Comment by daniellekirkwood** [comment]
@t3chguy would you update or close this issue? If it's still relevant it would benefit from a clearer description and then we can wrap it into our a11y work. In theory, it should be closed - we spent the last year working on Aria labels and focus.

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/element-hq/element-web/issues/22275
